sdk/anomalydetector/ai-anomaly-detector-rest/package.json

```diff
@@ -62,20 +62,19 @@
   "autoPublish": false,
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
-    "@azure-rest/core-client": "^1.0.0",
+    "@azure-rest/core-client": "^1.1.0",
     "@azure/core-rest-pipeline": "^1.8.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.2.0",
-    "@azure/core-paging": "^1.2.0",
-    "@azure/core-lro": "^2.2.0"
+    "@azure/core-paging": "^1.5.0"
   },
   "devDependencies": {
     "@microsoft/api-extractor": "^7.31.1",
     "autorest": "latest",
     "@types/node": "^14.0.0",
     "dotenv": "^16.0.0",
     "eslint": "^8.0.0",
-    "mkdirp": "^1.0.4",
+    "mkdirp": "^2.1.2",
     "prettier": "^2.5.1",
     "rimraf": "^3.0.0",
     "source-map-support": "^0.5.9",
```

Comment:
please update `typescript` dev dependency version to `~5.0.0`.  thanks!

Reply:
Done